### ISSUE TITLE
Font documentation should be internationalized

### ISSUE THREAD

**Issue by typiconman** [typiconman/fonts-cu#28]
It would be good to have documentation in Russian.

But right now I do not have time to work on it. This is a useful task if somebody else would like to get involved.

**Comment by aleslavista** [comment]
I don't know whether this may be of interest to you, but I can translate into Serbocroatian, just give me a couple of months to finalize the publication I'm working on at the moment.

**Comment by typiconman** [comment]
@aleslavista Yes, a translation into Serbo-Croation would be great.

**Comment by aleslavista** [comment]
OK, where's the file?

**Comment by typiconman** [comment]
Take a look here:
https://github.com/typiconman/fonts-cu/tree/master/docs

For an example of how to make multilingual documentation, take a look here:
https://github.com/slavonic/cu-tex/blob/master/churchslavonic.tex

**Comment by aleslavista** [comment]
Ahem, I'm sorry but I'm a LATEX illiterate, I know it exists and it has many backers but unfortunately I'm stuck to the stone age :(

**Comment by aleslavista** [comment]
If you don't mind I can translate the bare text, then you'll take care of typesetting it in LATEX.

**Comment by typiconman** [comment]
Yes, just make it in a plain text file and it'll import into TeX relatively easily. Will you do it in Cyrillic or Gaj?

**Comment by aleslavista** [comment]
OK, I'll just translate the English text, leaving the LATEX structure intact, that should make your job easier. I usually use Cyrillic, moreover, since we're talking about Cyrillic and Church Slavonic, I think Cirilica would be more suitable than Latinica. Unless you choose otherwise, of course.

**Comment by typiconman** [comment]
Yes, Cyrillic makes the most sense to me.

**Comment by aleslavista** [comment]
Right. As soon as I I'm finished with my book, I shall gladly take care of the translation.

**Comment by KrasnayaPloshchad** [comment]
Is it possible to intergrate online translating service as Crowdin or Transifex?

**Comment by aleslavista** [comment]
I uploaded a draft of the translation, it's just a rash translation, but before revising the draft I'd like you to make a PDF out of it first, to make sure I haven't broken LaTeX syntax. 

Also I'd like to know whether names of Graphite features like `Modern (Descender Right)`in Monomakh are actually just descriptions and therefore are to be translated.

Lastly if I may I suggest you use [Resavska BG](http://www.tipometar.org/aktuelno/akcija!/ResavskaBG/img/Resavska%20BG%20OTF.zip) from Typometar, it contains the specific Serbian glyphs and at the same time looks similar to the Roman font used in the original version.

[fonts-churchslavonic-sr.txt](https://github.com/typiconman/fonts-cu/files/1908788/fonts-churchslavonic-sr.txt)

**Comment by typiconman** [comment]
Thanks.
It does compile OK. You can also translate the keywords and a few other things. Regarding the Graphite features, their names are in English, and so should be left untranslated. Actually, there may be a way to localize the names of the Graphite features. I have to look into it.
A few changes need to be made to the TeX file for proper support of Serbo-Croation (for example, correct hyphenation), but I can do this myself. 

If you are able to use Git, then we could put this file up on GitHub and edit it together.
[fonts-churchslavonic-sr.pdf](https://github.com/typiconman/fonts-cu/files/1914561/fonts-churchslavonic-sr.pdf)

**Comment by typiconman** [comment]
Regarding the Resavska font: the only problem is that if we use a non-system font, we have to distribute it together with the documentation (since it's an Open Source project!). I don't know what font's license is and if it would be possible for us to distribute the font.

**Comment by KrasnayaPloshchad** [comment]
Linux Libertine has a special glyph for Serbian:
![default](https://user-images.githubusercontent.com/10364351/38824071-abfb1b50-41da-11e8-8aef-0ff6876da525.PNG)
Is it possible to present it in the document?

**Comment by typiconman** [comment]
This should turn on automatically when the document language is set to Serbian.

My question to @aleslavista: is there a problem with using Linux Libertine? Are there some characters needed for Serbo-Croation, for example, that are missing?

**Comment by KrasnayaPloshchad** [comment]
Oh I also remembered that Linux Libertine Italic has wrong glyph for ђ.

**Comment by typiconman** [comment]
They seem to have the glyphs needed for Serbian, but are missing many glyphs for Church Slavonic (and have some errors in a few glyphs as well). Regarding the italic ђ, @KrasnayaPloshchad, can you file a bug with Khaled on this?

**Comment by aleslavista** [comment]
@typiconman In my version of Libertine (I can't tell you the number but mind I'm running LO 5.4 on Lubuntu 17.10, so it must be one of the latest), upright BE doesn't have the Serbian glyph, italic does but it doesn't look convincing, plus kerning seems to be broken. Also italic TE lacks a Serbian glyph.

**Comment by typiconman** [comment]
Please file all of these as bugs against https://github.com/khaledhosny/libertinus

**Comment by aleslavista** [comment]
@typiconman Tipometar has notified me that their fonts can be freely used.

**Comment by aleslavista** [comment]
Updated version:
[fonts-churchslavonic-sr.txt](https://github.com/typiconman/fonts-cu/files/1936116/fonts-churchslavonic-sr.txt)

I've never heard of GitHub, please explain me how it works.

Concerning Graphite feature names, it seems they have two? names, or rather the longer one is simply a description? For instance, at the bottom of page 18: 

> "Truncation" feature ("trnc")

About Libertine, sorry to put it that way, but Cyrillic support is severely deficient, there are a myriad of issues, last but not least kerning, which make Libertine a very poor choice. Please consider Resavska BG, as I said above Tipmometar gives them away. Of course they should be thanked for it in the Credits section.

**Comment by typiconman** [comment]
@aleslavista Is the Serbo-Croation translation done? Can I merge it into master?

**Comment by aleslavista** [comment]
Hello Andreev, sorry if I'm late. Unfortunately at the moment I'm too busy, maybe this summer I'll be able to take care of translations.

**Comment by typiconman** [comment]
OK. Then I'll just merge what we have, and you can revise it later, if you have the time.

**Comment by aleslavista** [comment]
Please do not publish it now, it's full of mistakes. I promise I'll correct it as soon as possible, unfortunately at the moment I'm busy with lexicography.

**Comment by typiconman** [comment]
OK, but please don't close the issue. Maybe somebody else wants to work on it.

**Comment by aleslavista** [comment]
Sorry, I didn't mean to, I must have pressed the wrong button.

**Comment by aleslavista** [comment]
This is the almost definitive version: https://github.com/typiconman/fonts-cu/edit/update-dox/docs/fonts-churchslavonic-sr.tex We're almost done.

**Comment by aleslavista** [comment]
[Apparently there's no bug as I was mistaken in my understanding of date format in Serbian](https://github.com/reutenauer/polyglossia/issues/524#issuecomment-1040296839). Thus you may publish immediately: TeXworks though warns of some text overflows, but I just don't know how to fix them.
